tests/unit/test_utils.py

```diff
@@ -280,3 +281,47 @@ async def foo():
 
     exc_logs = [r for r in caplog.records if r.action == "foo_error" and r.exc_info]
     assert len(exc_logs) == 1
+
+
+def test_utcnow(mocker):
+    dt = mocker.patch("datetime.datetime")
+    assert utils.utcnow() == dt.now.return_value
+    dt.now.assert_called_once_with(datetime.timezone.utc)
+
+
+@pytest.mark.parametrize(
+    "offset, expected",
+    [
+        (None, datetime.datetime(2020, 1, 5, 0, 0, tzinfo=datetime.timezone.utc)),
+        (
+            2,
+            datetime.datetime(
+                2020, 1, 5, 0, 0, tzinfo=datetime.timezone(datetime.timedelta(hours=2))
+            ),
+        ),
+        (
+            -4,
+            datetime.datetime(
+                2020, 1, 5, 0, 0, tzinfo=datetime.timezone(datetime.timedelta(hours=-4))
+            ),
+        ),
+    ],
+)
+def test_aware_datetime(offset, expected):
+    assert utils.aware_datetime(2020, 1, 5, 0, 0, tz_offset=offset) == expected
+
+
+@pytest.mark.parametrize(
+    "input, expected",
+    [
+        ("2020-01-05", datetime.datetime(2020, 1, 5, 0, 0)),
+        (
+            "1984-06-02T19:05:57.720Z",
```

Comment:
Does it work with ` ` instead of `T` too ? And with `+01:00` ?

Reply:
It should, I've added more tests to showcase that ;)